Dockerfile

```diff
@@ -3,7 +3,7 @@
 ###
 
 # 1.) Get the Elixir dependencies within an Elixir container
-FROM hexpm/elixir:1.10.4-erlang-22.3.4.26-debian-buster-20210902 as elixir-builder
+FROM hexpm/elixir:1.12.3-erlang-22.3.4.26-debian-buster-20210902 as elixir-builder
```

Comment:
**question:** it looks like you upgraded the local version of Elixir in `.tool-versions` a couple of months ago, but I suppose this will actually update it when the site runs in the live production environment. Have you done any testing just to confirm there are no issues?

Reply:
good question! Before I had added this change, this PR failing the Docker build step was what actually tipped me off to this issue! 🤕 After adding this, I was successfuly able to deploy it to dev-green a few weeks ago.